src/components/ProgressIndicator/ProgressIndicator.stories.mdx

```diff
@@ -0,0 +1,288 @@
+import {
+  Meta,
+  Story,
+  ArgsTable,
+  Canvas,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import { Box, VStack } from "@chakra-ui/react";
+
+import ProgressIndicator from "./ProgressIndicator";
+import {
+  ProgressIndicatorSizes,
+  ProgressIndicatorTypes,
+} from "./ProgressIndicatorTypes";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("ProgressIndicator")}
+  component={ProgressIndicator}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=37638%3A23842",
+    },
+    jest: ["ProgressIndicator.test.tsx"],
+  }}
+  argTypes={{
+    id: { table: { disable: true } },
+  }}
+/>
+
+# ProgressIndicator
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.25.4`   |
+| Latest            | `0.25.4`   |
+
+<Description of={ProgressIndicator} />
+
+<Canvas withToolbar>
+  <Story
+    name="Basic with props"
+    args={{
+      darkMode: false,
+      indicatorType: ProgressIndicatorTypes.Linear,
+      isIndeterminate: false,
+      labelText: "Progress",
+      showLabel: true,
+      size: ProgressIndicatorSizes.Default,
+      value: 50,
+    }}
+  >
+    {(args) => <ProgressIndicator {...args} />}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Basic with props" />
+
+## Linear Type
+
+Progress bars are preferred in vertically narrow areas such as tables, cards,
+dialogs, etc.
+
+<Canvas>
+  <Story
+    name="Linear Type"
+    args={{
+      darkMode: false,
+      indicatorType: ProgressIndicatorTypes.Linear,
+      isIndeterminate: false,
+      labelText: "Linear Progress Type",
+      showLabel: true,
+      size: ProgressIndicatorSizes.Default,
+      value: 50,
+    }}
+    argTypes={{
+      indicatorType: { table: { disable: true } },
+    }}
+  >
+    {(args) => <ProgressIndicator {...args} />}
+  </Story>
+</Canvas>
+
+## Circular Type
+
+The circular progress type is preferred for large content areas and for
+full-screen loading. Set the `indicatorType` prop to
+`ProgressIndicatorTypes.Circular` for this type.
+
+<Canvas>
+  <Story
+    name="Circular Type"
+    args={{
+      darkMode: false,
+      indicatorType: ProgressIndicatorTypes.Circular,
+      isIndeterminate: false,
+      labelText: "Circular Progress Type",
+      showLabel: true,
+      size: ProgressIndicatorSizes.Default,
+      value: 50,
+    }}
+    argTypes={{
+      indicatorType: { table: { disable: true } },
+    }}
+  >
+    {(args) => <ProgressIndicator {...args} />}
+  </Story>
+</Canvas>
+
+### Sizing
```

Comment:
Are you using `###` for this and other headings because they are not listed in the sidebar menu?

Reply:
Yes, and because they're more variations and examples.